eclair-core/src/main/resources/reference.conf

```diff
@@ -84,6 +84,7 @@ eclair {
   // expiry-delta-blocks.
   fulfill-safety-before-timeout-blocks = 24
   min-final-expiry-delta-blocks = 30 // Bolt 11 invoice's min_final_cltv_expiry; must be strictly greater than fulfill-safety-before-timeout-blocks
+  max-block-processing-delay = 10 seconds // we add a random delay before processing blocks, capped at this value, to prevent herd effect
```

Comment:
I think a default of 30 seconds would be better if we want the effect to be really visible, don't you think? 10 seconds feels too small.

Reply:
```suggestion
  max-block-processing-delay = 30 seconds // we add a random delay before processing blocks, capped at this value, to prevent herd effect
```